ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -445,6 +473,19 @@ public void setSource(WebView view, @Nullable ReadableMap source) {
     view.loadUrl(BLANK_URL);
   }
 
+  @ReactProp(name = "urlSchemeBlacklist")
+  public void setUrlSchemeBlacklist(WebView view, @Nullable ReadableArray urlSchemeBlacklistArray) {
```

Comment:
Will this work when changing the urlBlacklistScheme from one non-null value to another non-null value?